webxrlayers-1.bs

```diff
@@ -492,18 +492,18 @@ interface XRWebGLLayerFactory {
 
   readonly attribute double nativeProjectionScaleFactor;
 
-  Promise&lt;XRProjectionLayer&gt; requestProjectionLayer(XRTextureType textureType,
-                                                   optional XRProjectionLayerInit init);
-  Promise&lt;XRQuadLayer&gt; requestQuadLayer(XRTextureType textureType,
-                                       XRLayerInit init);
-  Promise&lt;XRCylinderLayer&gt; requestCylinderLayer(XRTextureType textureType,
-                                               XRLayerInit init);
-  Promise&lt;XREquirectLayer&gt; requestEquirectLayer(XRTextureType textureType,
-                                               XRLayerInit init);
-  Promise&lt;XRCubeLayer&gt; requestCubeLayer(XRLayerInit init);
-
-  XRWebGLSubImage? getSubImage(XRLayer layer, XRFrame frame);
-  XRWebGLSubImage? getViewSubImage(XRLayer layer, XRView view);
+  XRProjectionLayer requestProjectionLayer(XRTextureType textureType,
```

Comment:
Ooh! One late comment! We've been using the term "request" exclusively with methods that return Promises till now. I wonder if this change means these should be called "create____Layer" instead?

Reply:
I like "create" better than "request". 
I will make that change.